android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -134,8 +134,13 @@ public int getTotalDiskCapacity() {
 
   @ReactMethod
   public int getFreeDiskStorage() {
-    StatFs external = new StatFs(Environment.getExternalStorageDirectory().getAbsolutePath());
-    return external.getAvailableBlocks() * external.getBlockSize();
+	try {
+		StatFs external = new StatFs(Environment.getExternalStorageDirectory().getAbsolutePath());
+		return external.getAvailableBlocks() * external.getBlockSize();
+	} catch (Exception e) {
+		e.printStackTrace();
+	}
+	return 0;
```

Comment:
Let's return `null` instead so that we can distinguish this from a "no space left"